### DISCUSSION TITLE
백준 1863 스카이라인 쉬운거 문제 추가 요청드립니다.

### DISCUSSION THREAD

**Discussion by cpprhtn** [tony9402/baekjoon#51] [문제 추가/수정 요청]
[선택] boj handle : xkzl9830

[필수] 문제 이름 : 스카이라인 쉬운거
[필수] 문제 번호 : 1863
[필수] 문제 링크 : https://www.acmicpc.net/problem/1863
[필수] Tag: 스택

**Comment by tony9402** [comment]
자료구조 태그에 추가하였습니다 :)
https://github.com/tony9402/baekjoon/tree/main/data_structure

**Reply by cpprhtn** [reply]
감사합니당 :D
